src/02-common-principles.md

```diff
@@ -559,52 +559,54 @@ for more information.
 
 ## The Inheritance Principle
 
-1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+In many BIDS datasets, any data file and its corresponding metadata file
+will possess an identical set of entities and suffix,
+differing only in their file extensions.
+This implicitly associates each data file with one metadata file,
+and one metadata file per data file.
+More advanced usages are however permissible:
+any data file may "inherit" metadata information from one or more metadata files,
+provided that the location of the metadata file in the file system
+makes it applicable to that data file.
+
+Where more than one metadata file is applicable to a data file,
+the contents of the file closest in filesystem position to the data file
+(lowest in the directory hierarchy, most entities in the file name)
+takes precedence.
```

Comment:
this is already a specific rule, which is incomplete (no description for order at the same level) and somewhat ambiguous -- "takes precedence" as the only file loaded, or loaded last (as what it is currently, but not explained here).  Thus I would just remove it from this "IP intro"

Reply:
> this is already a specific rule

Not sure what is intended here: it is an enumerated rule in the appendix, but the text here is intended as a more reader-friendly description.

> which is incomplete (no description for order at the same level)

This is tough to get the right balance between being reader-friendly and yet accurate. 

I'll try a complete alternative:
"Where more that one metadata file is applicable to the data file, all of those metadata files MUST be loaded, in a specific order dictated by the Inheritance Principle. This ordering influences the contents of sidecar information in the circumstance where the same key is defined in multiple files. That ordering is from the lowest to highest level in the filesystem hierarchy, with ordering from most to least number of entities within each directory."

Reply:
> but the text here is intended as a more reader-friendly description.

I do understand a motivation here, but IMHO it causes duplication and due to different wording thus potential ambiguity.   I do not think any rewording would *help* as it would keep that duplication in place.

That is why my suggestion is to remove this "reader-friendly" version (starting from "That ordering is from the highest...")  from here and just refer to a singular version in the appendix specification. Prior sentence might also want to be adjusted since I do not think BIDS defines "sidecar information" (metadata? ;)) .

Re "reader-friendly": some BIDS starter or blog posts could provide "user-friendly" description for IP if there is need, while leaving "specification" to provide condensed motivation (ideally) a singular clear version of how it works.  

Just my 1c.

Reply:
I think I like the idea of essentially "deferring" the reader to the IP appendix for any concept that is too difficult to convey in friendly text and/or runs the risk of imprecise duplication. I'll give this a try.

Reply:
See https://github.com/bids-standard/bids-specification/pull/1003/commits/c9c767d1e200f79c2b990127e5130cd3173eb678.